L: 0.7
lfw: 0.15
max_accel: 1
max_vel: 1
max_delta: rad(pi/6)
width: 0.3
length: 0.8
height: 0.15
sim_hz: 30
lookahead_dist: 1
lookahead_time: 0.1
voronoi_buffer: 0.5
